.github/build.yaml.gomplate

```diff
@@ -473,8 +473,6 @@
           sudo sed -i "s|<serial type='pty'>|<serial type='file'><source path='$GITHUB_WORKSPACE/serial_log.log'/>|" $(sudo find /usr/share -name "domain.xml.erb" -print -quit|xargs)
       - name: Prepare tests 🔧
         run: |
-          go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
Because we were already doing `go run github.com/onsi/ginkgo/v2/ginkgo` form the module folder in the Makefile and our `go.mod` includes all the stuff we need. For some reason I noted `go mod tidy` misses some ginkgo dependencies, hence this is what is manually added with `go get` in `go.sum`. In any case I think this is the best approach, we just used whatever is in go.mod and go with it.

Reply:
I think for this repository staying with fixed ginkgo and gomega should be fine, so we minimize the impact of third party libraries being updated at any point in time. Would be good if we only have to touch it when we plan for it or when it turn to be just too old and causes failures.